src/tests/ftest/io/mdtest_small.yaml

```diff
@@ -2,24 +2,33 @@ hosts:
   test_servers:
     - server-A
     - server-B
-    - server-C
-    - server-D
   test_clients:
     - client-E
     - client-F
 timeout: 1500
 server_config:
-    name: daos_server
+  servers_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      scm_mount: /mnt/daos1
 pool:
-    createmode:
-        mode_RW:
-             mode: 146
-    createset:
-        group: daos_server
-    createsize:
-        scm_size: 6000000000
-    createsvc:
-        svcn: 1
+  mode: 146
+  name: daos_server
+  scm_size: 5000000000
```

Comment:
Why reducing it?

Reply:
I hit this error in a previous run:

2020-01-19 01:43:43,839 test_utils_pool  L0132 INFO | Creating a pool
2020-01-19 01:43:47,298 test_utils_base  L0111 DEBUG| Exception raised by pydaos.raw.daos_api.create(svcn=1, group=daos_server, uid=1101, scm_size=6000000000, gid=1101, mode=146)
Traceback (most recent call last):
  File "/var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-1615/install/lib/daos/TESTING/ftest/util/test_utils_base.py", line 103, in _call_method
    method(**kwargs)
  File "/var/lib/jenkins/jenkins-1/docker_1/workspace/daos-stack_daos_PR-1615/install/lib64/python2.7/site-packages/pydaos/raw/daos_api.py", line 119, in create
    .format(ret))
DaosApiError: Pool create returned non-zero. RC: -1007